code/modules/assembly/motor.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/assembly/motor
+	name = "motor"
+	desc = "Used for upgrading the speed of various machinery."
+	icon_state = "motor"
+	custom_materials = list(/datum/material/iron=800, /datum/material/glass=200)
+	drop_sound = 'sound/items/handling/component_drop.ogg'
+	pickup_sound =  'sound/items/handling/component_pickup.ogg'
```

Comment:
there is no reason for now other than "its really tedious" since those are non stackable. but if you think it would be better balancing i could either make it heat up the gas or something else like a probability of spewing gas

Reply:
Make it eat more power.

Reply:
power is not an effective balancer until power is actually fixed, so that's a non starter

Reply:
Doesn't mean it shouldn't be done though.